news_items/005_fractions_and_input_changes/index.html

```diff
@@ -0,0 +1,81 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS '21)
+  has added some big improvements to PrairieLearn.
+</p>
+
+<h5>Fraction input</h5>
+
+<p>
+  One of the most-requested features ever! You can now enter fractions
+  instead of numbers:
+</p>
+
+<img src="approx_pi.png" width="600px">
+
+<p>
+  You can even enter fractions with decimal points, like <tt>10.4/12.6</tt>
+</p>
+
+
+
+<h5>Clear handling of invalid answers</h5>
+
+<p>
+  If you submit an answer that PrairieLearn can't understand,
+  like <code>asf#$%J34jkl</code> when it's expecting a number, then it
+  will now show you clearly that something is wrong:
+</p>
+
+<img src="invalid_input.png" width="260px">
+
+<p>
+  Best of all, PrairieLearn doesn't count invalid answers as
+  "attempts", so you don't lose any points!
+</p>
+
+<p>
+  Clicking on the <span class="badge
+  badge-secondary">Why <i class="fas fa-question-circle"></i></span>
+  button will give more information on why the answer couldn't be
+  understood, including some example answers that are valid and
+  invalid:
+</p>
+
+<img src="why_btn.png" width="476px">
+
+
+
+
+<h5>Other improvements</h5>
+
+<p>
+  Previously, questions could fail if the instructor changed the
+  question code after you started working. Now PrairieLearn will show
+  a "Missing Input" label and the question will continue to work:
+</p>
+
+<img src="missing_input.png" width="256px">
+
+<p>
+  Inputs with "tall" labels will now correctly scale:
+</p>
+
+<img src="full_height_input.png" width="494px">
+
+
+
+<hr class="mt-5">
+<p class="text-right small">
```

Comment:
I think this should be the default for all news article footers?

Reply:
Not a bad idea, I'll leave it manually coded for now but we can add later to the PL news item code itself.